package.json

```diff
@@ -28,10 +28,8 @@
     "@types/geojson": "^1.0.6",
     "@types/mapbox-gl": "^0.51.4",
     "chroma-js": "^1.3.5",
-    "mapbox-gl": "1.4.0",
+    "mapbox-gl": "^1.0.0",
```

Comment:
For consistent dependency versions, we should add `package-lock.json` to version control.

A default `npm install` will get 1.4.1, but if you have an existing `package-lock.json` with an older version, you need to delete the package-lock file and node_modules directory first.